.coafile

```diff
@@ -42,6 +42,10 @@ enabled = False
 bears = KeywordBear
 language = python3
 
+[PythonPackageInit]
+files = bears/**.py
+bears = PythonPackageInitBear
+
 # Note that the ci_keywords and cs_keywords are only used here because coala
 # 0.8.x (current stable release) needs them.
 ci_keywords, keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
```

Comment:
Oh... I misunderstood earlier, as of this commit am moving it into `python` section, I don't think there's any need of making another one